temp/harmonic_analysis.py

```diff
@@ -0,0 +1,106 @@
+"""Pseudocode for PyDPF-Post Harmonic Mechanical Simulations APIs."""
+
+import ansys.dpf.post as dpf
+from ansys.dpf.post import examples
+from ansys.dpf.post.common import AvailableSimulationTypes
+from ansys.dpf.post.selection import Selection
+
+# Load the simulation files
+harmonic_simulation = dpf.load_simulation(
+    examples.complex_rst, simulation_type=AvailableSimulationTypes.harmonic_mechanical
+)
+# # Also possible:
+# harmonic_simulation = dpf.load_simulation(examples.complex_rst)
+# harmonic_simulation = dpf.load_simulation(
+#     examples.complex_rst, simulation_type="harmonic mechanical"
+# )
+
+# -----------------------------------------------------------------------------------------
+# Explore the simulation metadata
+
+# Print information about the simulation
+print(harmonic_simulation)
+
+# Print the mesh
+print(harmonic_simulation.mesh)
+
+# Print the list of constructed geometries
+print(harmonic_simulation.constructed_geometries)
+
+# Print the list of boundary conditions
+print(harmonic_simulation.boundary_conditions)
+
+# Print the list of available named selections
+named_selections = harmonic_simulation.named_selections
+print(named_selections)
```

Comment:
```suggestion
print(harmonic_simulation.named_selections)
```

Reply:
@MaxJPRey `named_selections` is used further down in the code.

Reply:
OK.
This suggestion is to be discarded.